src/message/__tests__/fetchActions-test.js

```diff
@@ -82,7 +83,10 @@ describe('fetchActions', () => {
 
   describe('tryFetch', () => {
     test('resolves any promise, if there is no exception', async () => {
-      const tryFetchFunc = () => new Promise(resolve => setTimeout(() => resolve('hello'), 100));
+      const tryFetchFunc = async () => {
+        await sleep(10);
```

Comment:
FWIW, Re: the commit message, I don't think 10ms is a very realistic API request response time, although I don think it's reasonable to use in a test. You'd need to be quite close to the server, and calling a fast API endpoint to get 10ms.

Reply:
Yeah; I see that I didn't back up that statement with anything. 🤔 I think I was going for "more realistic than no sleep at all (what the error case was doing) but still wouldn't slow down the tests as noticeably as 100ms." I'll fix it.

Could you please confirm that you meant "do" instead of "don"?

Reply:
Looks like `GET /server_settings` on CZO is taking ~300ms for me.

Reply:
Ah, yeah, I meant "do" — 10ms seems fine, it's hard to imagine what wouldn't break at 10ms but would at 100ms.